### DISCUSSION TITLE
我在雨云的服务器上面，搭建了此面板，却无法在手机端上访问（可以打开 就一直卡在加载界面），PC端正常访问

### DISCUSSION THREAD

**Discussion by moshao1128** [1Panel-dev/1Panel#5809] [Q&A]
### 1Panel 版本

v1.10.11-lts

### 问题描述

我在雨云的服务器上面，搭建了此面板，却无法在手机端上访问（可以打开 就一直卡在加载界面），PC端正常访问，无论是在雨云自带的预览这里安装，还是在手动安装，都一直反复出现此问题，多次重装系统都无法成功修复问题，手机打开电脑模式也无法解决

### 重现步骤

打开雨云服务器面板，进入预览应用安装的面板 卡在加载界面（手机端）
打开雨云服务器面板，按照dp次元星球（b站的阿婆主，主要步骤还是以官方文档为主）的搭建步骤搭建此面板，进入卡在加载界面（手机端）

### 期待的正确结果

手机正常访问，雨云服务器上

**Comment by wanghe-fit2cloud** [comment]
`PC 端正常访问` 说明 1Panel 服务的运行状态没有问题。至于移动端无法访问的问题，可以先自行检查网络和端口等因素。
